azure/notebooks/kubernetes/Dask_Kube_XGBoost.ipynb

```diff
@@ -0,0 +1,917 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    xgb_cuda_model = xgboost_output['booster']
</pre><p>let's rename to xgb_gpu_model (not expecting all the data science users to be familiar with our software stack)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Alright. </p>